chameleon.py

```diff
@@ -68,20 +90,28 @@ def stop():
 def _get_cherrypy_page_handler(status_code, content_type, content):
     class Foo(object):
         @cherrypy.expose
+        @cherrypy.tools.json_out()
```

Comment:
It deals fine with strings, I have that in the tests. Not sure about XML

My understanding is that it might be activated rather than will be.
I could be wrong.